dev/conductor/core/bin/packages_autoroller.dart

```diff
@@ -17,12 +18,21 @@ const String kGithubClient = 'github-client';
 const String kMirrorRemote = 'mirror-remote';
 const String kUpstreamRemote = 'upstream-remote';
 
-Future<void> main(List<String> args) async {
+Future<void> main(List<String> args) {
+  return run(args);
+}
+
+@visibleForTesting
+Future<void> run(
+  List<String> args, {
+  FileSystem fs = const LocalFileSystem(),
+  ProcessManager processManager = const LocalProcessManager(),
+}) async {
   final ArgParser parser = ArgParser();
   parser.addOption(
     kTokenOption,
-    help: 'GitHub access token env variable name.',
-    defaultsTo: 'GITHUB_TOKEN',
+    help: 'Path to GitHub access token file.',
```

Comment:
by reading the token from a file, the actual token should not show up in CI logs